server/create.go

```diff
@@ -46,6 +49,13 @@ func (s *Server) CreateHandler(c *gin.Context) {
 		return
 	}
 
+	for f := range maps.Keys(r.Files) {
```

Comment:
```suggestion
	for f := range r.Files {
```

is there an advantage of `maps.Keys` ?

Reply:
Nope, that was just a miss on my part, fixed